lib/editor/components/timetable/TripSeriesModal.js

```diff
@@ -97,12 +98,24 @@ const TripSeriesModal = (props: Props) => {
         if (currentBlockIdSuffix > autoBlockIdIncrement) currentBlockIdSuffix = currentBlockIdSuffix % autoBlockIdIncrement
       }
     }
-    clearState()
+    setState(initialState)
     onClose()
-  }, [endTime, startTime, headway, autoTripIdStart, autoTripIdIncrement, autoTripIdPrefix, autoBlockIdPrefix, autoBlockIdIncrement, autoBlockIdStart])
+  }, [state])
```

Comment:
this is a scary one, because this will now fire if _any_ state variable changes, not just the ones we want. Maybe we should make it explicit? `state.endTime, state.starTime`... and so on

Reply:
hmmmm

Reply:
Miles is right, there are a couple state variables that we don't want to fire on, it would technically be best to set this up explicitly.

Reply:
good points